vectordatasource/meta/sql.py

```diff
@@ -167,6 +167,8 @@
     'mz_get_min_zoom_highway_level_gate': 'mz_get_min_zoom_highway_level_gate',
     'util.calculate_path_major_route':    'mz_calculate_path_major_route',
     'mz_calculate_ferry_level':           'mz_calculate_ferry_level',
+    'util.is_building':                   'mz_calculate_is_building_or_part',
+    'util.tag_yes_to_bool':               'notimplemented',
```

Comment:
I think it's only necessary if the function is used for calculating the `min_zoom`. Otherwise we can omit it, and it'll only cause problems if we try to use the PostgreSQL-side properties calculation. I don't think we plan on using that again... it's just hanging around for completeness' sake, right?